src/components/MajorList/index.js

```diff
@@ -13,14 +13,29 @@ export default class Majors extends Component {
   render() {
     let content;
     let majorPrefix;
+    // Gets the row item widths
+    let rowWidths = this.props.rowWidths.twoItems;
+    let rowItemOne = rowWidths.itemOne;
+    let rowItemTwo = rowWidths.itemTwo;
+
+    // Creates the list item's content
     if (this.props.majors) {
-      content = this.props.majors.map(major => (
-        <div>
-          <Typography>{major}</Typography>
-        </div>
-      ));
+      // Gets the last item of the list
+      let lastItem = this.props.majors[this.props.majors.length - 1];
+
+      // If there are multiple majors, each major name will have a comma after it except for the
+      // very last. If there's only one major, no comma will appear
+      let majorText = '';
+
+      this.props.majors.forEach(major => {
+        // NOTE: The difference between the two statements that adds text to the variable
+        // "majorText" is that one of them adds BOTH a comma and a space
+        major === lastItem ? (majorText += `${major}`) : (majorText += `${major}, `);
+      });
+      content = <Typography>{majorText}</Typography>;
```

Comment:
Updates the way a student's majors are displayed. Before, if the student had multiple majors, their major names would appear in a column. This creates a lot of white space due to multiple lines being created. This has been changed so that the major names are on the same line and are separated by a comma. If there's only one major, no comma will display.